modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -406,6 +415,40 @@ public ObjectEntry putObjectEntry(
 			objectEntryId, objectEntry);
 	}
 
+	@Override
+	public void putObjectEntryObjectActionObjectActionName(
+			Long objectEntryId, String objectActionName)
+		throws Exception {
+
+		com.liferay.object.model.ObjectEntry serviceBuilderObjectEntry =
+			_objectEntryLocalService.getObjectEntry(objectEntryId);
+
+		_objectActionEngine.executeObjectAction(
```

Comment:
I was thinking about this... As this is an endpoint to connect with external systems, I supposed that we should check if the user that is doing the request has permission to execute the actions, right?
Is this method checking permissions?
Because what I'm seeing [here](https://github.com/liferay-headless/liferay-portal/pull/867/commits/731a9ce051656dc04f82e174114d4d715003a3be#diff-4f5febc18703eb319846ac0460606b972c23cc409cc146f37d0adae7a382a270R58-R79) is that we are getting all the information from the local services and the method [`getObjectActionExecutor`](https://github.com/liferay/liferay-portal/blob/a0f7427be0f5dfc8ba9dfc4824105d709f63b8af/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/ObjectActionExecutorRegistryImpl.java#L40-L52) gets the `ObjectActionExecutor` from a `_serviceTrackerMap` 
Am I missing something?

Sorry I didn't point this at my first review 😓

Reply:
Thanks a lot for your review @LuismiBarcos! It helped a lot 😄 

I had added a `getObjectEntry` call [here](https://github.com/liferay-headless/liferay-portal/blob/17d1778b37607a74686a5a6298917a1b8db44d97/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java#L442-L445), which checks the `VIEW` permission for the `ObjectEntry`.
 
But I'll confirm this behavior with the team and resend this PR.